aspics/params.py

```diff
@@ -16,39 +15,63 @@ class Params:
     """Convenience class for setting simulator parameters. Also holds the default values."""
 
     def __init__(
-        self,
-        location_hazard_multipliers=LocationHazardMultipliers(
-            retail=0.0165,
-            nightclubs=0.0165,
-            primary_school=0.0165,
-            secondary_school=0.0165,
-            home=0.0165,
-            work=0.0,
-        ),
-        individual_hazard_multipliers=IndividualHazardMultipliers(
-            presymptomatic=1.0, asymptomatic=0.75, symptomatic=1.0
-        ),
-        obesity_multipliers=[1, 1, 1, 1],
-        cvd_multiplier=1,
-        diabetes_multiplier=1,
-        bloodpressure_multiplier=1,
-        overweight_sympt_mplier=1.46,
+            self,
+            ### PLACES#######
+            location_hazard_multipliers=LocationHazardMultipliers(
+                retail=0.0165,
+                primary_school=0.0165,
+                secondary_school=0.0165,
+                home=0.0165,
+                work=0.0,
+            ),
+            ### DISEASE STATUS##########
+            individual_hazard_multipliers=IndividualHazardMultipliers(
+                presymptomatic=1.0, asymptomatic=0.75, symptomatic=1.0
+            ),
+            ###########################################
+            #####Health Conditions (Type, BMI) ########
+            ###########################################
+            obesity_multipliers=[1, 1, 1, 1],
+            sex_multipliers = [1,1,1,1],
+            ethnicity_multipliers =[1,1,1,1],
+            age_multipliers = [1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],
```

Comment:
How about we do something like `[1 for _ in range(4)]` where 4 is the number of things we need? Kind of like `np.ones(4)`